py-rse/package-py.Rmd

```diff
@@ -45,20 +70,11 @@ pkg_name
 └── setup.py
 ```
 
-which in our case would become
-
-```text
-zipfs_law
-├── zipfs_law
-│   ├── collate.py
-│   ├── countwords.py
-│   ├── mymodule.py
-│   └── plotcounts.py
-├── README.md
-└── setup.py
-```
+which in our case would mean changing the name of `bin/` to `zipfs_law/`.
 
-It may initially be confusing to have two folders with the same name,
+It may initially be confusing to have two folders with the same name
+(the original project directory,
+and the renamed `bin/` containing `*.py`),
```

Comment:
removed "containing '*.py'" maybe - I don't think it adds?

Reply:
yes!